tests/components/bmw_connected_drive/__init__.py

```diff
@@ -147,3 +51,58 @@ async def setup_mocked_integration(hass: HomeAssistant) -> MockConfigEntry:
     await hass.async_block_till_done()
 
     return mock_config_entry
+
+
+def check_remote_service_call(
+    router: respx.MockRouter,
+    remote_service: str = None,
+    remote_service_params: dict = None,
+    remote_service_payload: dict = None,
+):
+    """Check if the last call was a successful remote service call."""
+
+    # Check if remote service call was made correctly
+    if remote_service:
+        # Get remote service call
+        first_remote_service_call: respx.models.Call = next(
+            iter(
+                [
```

Comment:
Just make a generator expression instead. We don't need to first make a list and then use `iter`.

Reply:
Thanks for the hint. Always learning new things :)